pkg/scheduler/scheduler_test.go

```diff
@@ -425,3 +426,78 @@ func getPodFromPriorityQueue(queue *internalqueue.PriorityQueue, pod *v1.Pod) *v
 
 	return nil
 }
+
+func TestInitPluginsWithIndexers(t *testing.T) {
+	name := "AddIndexer"
+	entrypoint := func(obj runtime.Object, handle framework.Handle) (framework.Plugin, error) {
+		podInformer := handle.SharedInformerFactory().Core().V1().Pods()
+		podInformer.Informer().GetIndexer().AddIndexers(cache.Indexers{
+			"nodeName": indexByPodNodeName,
+		})
+		return &TestPlugin{name: name}, nil
+	}
+
+	fakeInformerFactory := NewInformerFactory(&fake.Clientset{}, 0*time.Second)
+
+	registerPluginFuncs := []st.RegisterPluginFunc{
+		// anything supported by TestPlugin is fine
+		st.RegisterFilterPlugin(name, entrypoint),
+		// we always need this
+		st.RegisterQueueSortPlugin(name+"QueueSort", newQueueSortPlugin),
+		st.RegisterBindPlugin(name+"Bind", newBindPlugin),
```

Comment:
You can reuse the default queuesort and default bind plugins:

```go
st.RegisterQueueSortPlugin(queuesort.Name, queuesort.New),
st.RegisterBindPlugin(defaultbinder.Name, defaultbinder.New),
```

Reply:
done, thanks!